hack/configapi/e2e/test.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Try to upload a file to S3 and then delete it using the configapi cli.
+# Check the file exists after uploading it.
+# Check the file does not exist after deleting it.
+
+readonly REGION="eu-central-1"
+readonly BUCKET="cdn-constellation-backend"
+# readonly BUCKET="resource-api-testing"
+
+readonly CLAIM_PATH="${TEST_TMPDIR}/maaClaim.json"
+cat << EOF > "$CLAIM_PATH"
+{
+  "x-ms-isolation-tee": {
+    "x-ms-sevsnpvm-tee-svn": 1,
+    "x-ms-sevsnpvm-snpfw-svn": 9,
+    "x-ms-sevsnpvm-microcode-svn": 116,
+    "x-ms-sevsnpvm-bootloader-svn": 4
+  }
+}
+EOF
+
+readonly DATE="2023-02-02-03-04"
+${PATH_CLI} --maa-claims-path "$CLAIM_PATH" --upload-date "$DATE" --region "$REGION" --bucket "$BUCKET"
+
+sleep .2 # Ensure cache invalidation has time to propagate.
```

Comment:
I had problems with the curl still picking up the file after deletion. The sleep resolved it. Didn't investigate further, so the reason may be another one.

Reply:
I think we should investigate this and if the CLI is returning to early rather sleep in the Go code than after each invocation.

Reply:
We should extend the close invocation of the CLI a bit in the style of https://github.com/edgelesssys/constellation/blob/main/internal/api/versionsapi/cli/rm.go#L108-L113 to get better feedback and fail in case we cannot successfully invalidate the CDN path.